### ISSUE TITLE
[Bug] Docker failing to build due to pdal version requirement

### ISSUE THREAD

**Issue by snlawrence** [OSGeo/grass#2400]
**Describe the bug**
Docker container fails to build due to pdal version requirement. Version >=1.7.1 is required in the configure stage and the installed version is 1.6.0

**To Reproduce**

1. Clone repository
2. Run `docker build -t grassgis .`
3. Build fails with the following error:
```
configure: error: *** Unable to locate suitable (>=1.7.1) PDAL library.
The command '/bin/sh -c ./configure     --enable-largefile     --with-cxx     --with-nls     --with-readline     --with-sqlite     --with-bzlib     --with-zstd     --with-cairo --with-cairo-ldflags=-lfontconfig     --with-freetype --with-freetype-includes="/usr/include/freetype2/"     --with-fftw     --with-netcdf     --with-pdal     --with-proj --with-proj-share=/usr/share/proj     --with-geos=/usr/bin/geos-config     --with-postgres --with-postgres-includes="/usr/include/postgresql"     --with-opengl-libs=/usr/include/GL     && make -j $NUMTHREADS && make install && ldconfig' returned a non-zero code: 1
```

**Expected behavior**
Docker container builds successfully.

**System description (please complete the following information):**

- Operating System: Linux CentOS 8 Stream
- GRASS GIS version: 8.0.2

**Comment by neteler** [comment]
Perhaps the EPEL repo needs to be added for CentOS 8?

https://centos.pkgs.org/8/epel-x86_64/PDAL-2.1.0-8.el8.x86_64.rpm.html

**Comment by snlawrence** [comment]
I don't think that should make a difference because I'm building inside the docker image, so it should be independent of my system. (I did add epel to my Centos 8 just to be sure, but the same error occurs when building the docker image)

**Comment by neteler** [comment]
I have checked with `git blame`, the minimum version of PDAL 1.7 has been introduced in #1200.

See also this comment: https://github.com/OSGeo/grass/pull/1200#issuecomment-864222306

**Comment by marisn** [comment]
Can't comment on the Docker part but going to anything less than PDAL 2.0 is a waste of time. --without-pdal is always an option (at a cost of r|v.in.pdal).
